lib/grppc.gd

```diff
@@ -519,6 +519,18 @@ DeclareGlobalFunction(
 ##
 DeclareGlobalFunction( "GapInputPcGroup" );
 
+#############################################################################
+##
+#F  PrintPcPresentation( <grp>, <commBool> )
+##
+##  <#GAPDoc Label="PrintPcPresentation">
+##  <ManSection>
+##  <Func Name="PrintPcPresentation" Arg='grp, commBool'/>
+##  </ManSection>
+##  <#/GAPDoc>
```

Comment:
Note that this GAPDoc comment won't appear in the manual, as it would have to be referenced there explicitly. But that's fine, as we do not want to explicitly document this function. I just wanted to point this out -- the above 5 lines could be safely deleted.

But *if* we keep them, it is weird that the actual content which would normally go inside the `<Description>` of this `<ManSection>` is not here, but rather in the `.gi` file. That seems to be neither here nor there. So I'd either remove the GAPDoc here; or else move the actual description text from the `.gi` file into this comment.

Reply:
Thanks for the replay. I have no idea how GAPDoc works and just tried to do the "necessary" documentation by (educated) copy and paste. I'm just deleting these lines.